lib/game.rb

```diff
@@ -0,0 +1,97 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2, :current_player
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+  ]
+
+  def initialize( player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    turns = @board.turn_count
+    # binding.pry
+    if turns % 2 == 0
+      @current_player = @player_1
+    else 
+      @current_player = @player_2
+    end
+    return @current_player
```

Comment:
Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`

Notice the implicit return values to.